packages/core/src/2d/sprite/SpriteRenderer.ts

```diff
@@ -93,6 +99,28 @@ export class SpriteRenderer extends Renderer {
     }
   }
 
+  /**
+   * Interacts with the masks.
+   */
+  get maskInteraction(): SpriteMaskInteraction {
+    return this._maskInteraction;
+  }
+
+  set maskInteraction(value: SpriteMaskInteraction) {
+
+  }
+
+  /**
+   * The mask layer the sprite renderer belongs to.
+   */
+  get maskLayer(): number {
```

Comment:
return type is enum

Reply:
> return type is enum

Number is better，SpriteMaskLayer.Layer0 | SpriteMaskLayer.Layer1